azext_iot/operations/central.py

```diff
@@ -43,19 +56,21 @@ def iot_central_validate_messages(
     yes=False,
     central_api_uri="api.azureiotcentral.com",
 ):
+    provider = CentralDeviceProvider(cmd, app_id)
     _events3_runner(
-        cmd,
-        app_id,
-        device_id,
-        True,
-        simulate_errors,
-        consumer_group,
-        timeout,
-        enqueued_time,
-        repair,
-        properties,
-        yes,
-        central_api_uri,
+        cmd=cmd,
+        app_id=app_id,
+        device_id=device_id,
```

Comment:
thanks, using keyword args is good practice when the argument list grows unwieldly.